ocaml/xapi/create_misc.ml

```diff
@@ -470,7 +470,8 @@ let create_root_user ~__context =
     Db.User.create ~__context ~ref ~fullname ~short_name ~uuid ~other_config:[]
 
 let get_xapi_verstring () =
-  Printf.sprintf "%d.%d" Constants.version_major Constants.version_minor
+  Printf.sprintf "%d.%d.%s" Constants.version_major Constants.version_minor
```

Comment:
Not sure whether we should put the extra version info into this field or into a separate field, if there is any software out there that parsed this version number and expected a 2 numbers it'd break.
It is fine to change this on trunk, but if we want to backport it we should probably add a separate full-xapi-version field there?